amplify-tools-gradle-plugin/src/main/groovy/com/amplifyframework/tools/gradle/plugin/AmplifyTools.groovy

```diff
@@ -0,0 +1,219 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class AmplifyTools implements Plugin<Project> {
+    void apply(Project project) {
+        def doesNodeExist = true
+        def doesGradleConfigExist
+
+        def profile = 'default'
+        def accessKeyId = null
+        def secretAccessKey = null
+        def region = null
+        def envName = 'amplify'
+        def syncEnabled = 'true'
+
+        project.task('verifyNode') {
+            try {
+                project.exec {
+                    commandLine 'which', 'node'
```

Comment:
Seems like we need to make a simple change here to support Window. We can probably detect Windows using https://docs.gradle.org/current/javadoc/org/gradle/nativeplatform/platform/OperatingSystem.html and run a different command.

Reply:
It may also be that ```which``` (not a valid command in cmd) is causing the problem. Testing this out using ```node -v``` in windows

Reply:
Nice. Did this work? Do we have a Windows compatible IDE integration now? 😀

Reply:
Tested it out after a lot of struggles with windows, and got the build working! We need to use the OS detection to switch between npx.cmd and npx as well as amplify.cmd and amplify commands, and use node -v instead of which

Reply:
NICE!